terraform/modules/happy-service-eks/variables.tf

```diff
@@ -137,6 +137,18 @@ variable "period_seconds" {
   description = "The period in seconds used for the liveness and readiness probes."
 }
 
+variable "liveness_timeout_seconds" {
+  type        = number
+  default     = 30
+  description = "Timeout for liveness probe."
+}
+
+variable "readiness_timeout_seconds" {
+  type        = number
+  default     = 30
+  description = "Readiness probe timeout seconds"
+}
```

Comment:
I think you'll also want to add these to happy-stack-eks since that's the top-level module

Reply:
Done! Added to stack variables and main.tf service definition.